RELEASE/scripts/autoscend/paths/avatar_of_jarlsberg.ash

```diff
@@ -0,0 +1,146 @@
+boolean isJarlsberg()
+{
+	return (my_class() == $class[Avatar of Jarlsberg] || my_path() == "Avatar of Jarlsberg");
+}
+
+void jarlsberg_initializeSettings()
+{
+    	if(isJarlsberg())
+	{
+		auto_log_info("Initializing Avatar of Jarlsberg settings", "blue");
+		set_property("auto_jarlsbergSkills", -1);
```

Comment:
instead of resetting `auto_jarlsbergSkills` when initializing, you should just make it `_auto_jarlsbergSkills`
mafia automatically deletes all properties that starts with _ on dayroll or on ascension.

Reply:
This section was modelled on the other avatar paths. As I understand it `jarlsberg_initializeSettings` (and equivalents for Boris/Pete/Ed) are only called at the start of the ascension. Don't think the number of skills autoscend thinks we have resetting each day would actually cause many (if any) problems, but not sure we want it to happen?

However, if we do change this one to `_auto_jarlsbergSkills`, should the Boris/Pete/Ed etc equivalents also be changed for consistency?

Reply:
ideally they should be changed as well but it does not have to be in this PR.